nautobot_ssot/jobs/examples.py

```diff
@@ -943,6 +947,38 @@ def data_mappings(cls):
             DataMapping("Interface (local)", reverse("dcim:interface_list"), "Interface (remote)", None),
         )
 
+    def run(  # pylint: disable=too-many-arguments, arguments-differ
+        self,
+        dryrun,
+        memory_profiling,
+        target,
+        target_url,
+        target_token,
+        *args,
+        **kwargs,
+    ):
+        """Run sync."""
+        self.dryrun = dryrun
+        self.memory_profiling = memory_profiling
```

Comment:
As long as we're using celery it's not safe to store any information on the Job instance

Reply:
This has been required for all SSoT Apps as we need to save the value and pass it to super().

Reply:
It doesn't have to be fixed here but since celery only instantiates a task once it's dangerous to store anything on the instance. That is probably something that should be followed up on

Reply:
Also related to the secrets being stored on the instance below, credentials could be leaked between jobs using this pattern.

Reply:
@gsnider2195 I'll follow up with you about this.